keyboards/kbdfans/kbd67/rev2/keymaps/droxx/rules.mk

```
@@ -0,0 +1,17 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys (+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        	# Console for debug (+400)
+COMMAND_ENABLE = no        	# Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      	# Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard underglow functionality
+MIDI_ENABLE = no            # MIDI controls (+2400 to 4200)
+AUDIO_ENABLE = no           # Enable the audio subsystem
+BLUETOOTH_ENABLE = no		# Enable Bluetooth with the Adafruit EZ-Key HID
+
+# Do not enable SLEEP_LED_ENABLE. It uses the same timer as BACKLIGHT_ENABLE.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```

This really should only be what is different. 
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
BACKLIGHT_ENABLE = no      	# Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard underglow functionality
```